android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -308,6 +312,37 @@ class OkHttpTest {
     assertEquals(sessionIds[0], sessionIds[1])
   }
 
+  @Test
+  fun testDnsOverHttps() {
+    assumeNetwork()
```

Comment:
This might be better than ignoring, since it will short circuit if no external network access.

Reply:
Nice.